documentation/content/en/books/handbook/bsdinstall/_index.adoc

```diff
@@ -541,13 +541,13 @@ Configuring too little swap can lead to inefficiencies in the VM page scanning c
 
 On larger systems with multiple SCSI disks or multiple IDE disks operating on different controllers, it is recommended that swap be configured on each drive, up to four drives.
 The swap partitions should be approximately the same size.
-The kernel can handle arbitrary sizes but internal data structures scale to 4 times the largest swap partition.
+The kernel can handle arbitrary sizes, but internal data structures scale to 4 times the largest swap partition.
 Keeping the swap partitions near the same size will allow the kernel to optimally stripe swap space across disks.
 Large swap sizes are fine, even if swap is not used much.
 It might be easier to recover from a runaway program before being forced to reboot.
 
 By properly partitioning a system, fragmentation introduced in the smaller write heavy partitions will not bleed over into the mostly read partitions. 
```

Comment:
Might as well use "write-heavy" here while you're in the neighborhood.

Reply:
Done, thank you.